src/views/Results.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div v-bind:class="layoutClass">
+    <div v-if="showSidebar" class="col1q">
+      <Facet v-for="facet in facets" v-bind:facet="facet" v-bind:key="facet" />
+    </div>
+    <div v-bind:class="contentClass">
+      <Item
+        v-for="(result, index) in results"
+        v-bind:result="result"
+        v-bind:index="index"
+        v-bind:key="result.id"
+      />
+      <Pagination v-if="showPagination" msg="Pagination bar" />
+    </div>
+  </div>
+</template>
+
+<script>
+import Facet from "@/components/Facet.vue";
+import Item from "@/components/Item.vue";
+import Pagination from "@/components/Pagination.vue";
+
+export default {
+  name: "Results",
+  components: {
+    Facet,
+    Item,
+    Pagination
+  },
+  data() {
+    return {
+      facets: [],
+      hits: 0,
+      results: [],
+      results_per_page: process.env.VUE_APP_RESULTS_PER_PAGE || 5,
+      status: {
+        error_message: "",
+        errored: false,
+        loading: false,
+        ready: false
```

Comment:
I don't see where `ready` is used for anything.

Reply:
The `ready` code here may not be used - I'll take it out.